packages/story-editor/src/components/dropTargets/karma/dropTarget.karma.js

```diff
@@ -349,9 +349,9 @@ describe('Drop-Target integration', () => {
           await fixture.events.mouse.clickOn(bgElement, 5, 5);
 
           // And flip it
-          await fixture.events.click(fixture.editor.inspector.designTab);
+          await fixture.events.click(fixture.editor.sidebar.designTab);
```

Comment:
🤔  Should the `designTab` be renamed to `styleTab`?

Reply:
I thought we are okay with leaving the Library (now Insert) and Design (now Style) as is, it was brought up [here](https://github.com/GoogleForCreators/web-stories-wp/pull/10760#discussion_r817626896). Do we want to change Design to Style in just some of the places (Fixture, component folder name)?

Reply:
Sam is sick today so we can do a follow-up PR on this if needed.